src/platforms/dotnet/common/unit-testing.mdx

```diff
@@ -1,22 +1,43 @@
 ---
-title: Unit Testing
-description: "Learn about unit testing with `ISentryClient` and `IHub`."
+title: Compatibility and Unit Testing
+sidebar_order: 20
+description: "Learn about compatiblity and unit testing with `ISentryClient` and `IHub`."
 ---
 
-We often don't want to couple our code with a static class like `SentrySdk`. Especially to allow our code to be testable.
-If that's your case, you can use 2 abstractions:
+According to this [compatibility table](https://docs.microsoft.com/en-us/dotnet/standard/net-standard), the package is compatible with the following versions or newer:
 
-- `ISentryClient`
-- `IHub`
+- .NET Framework 4.6.1
+- .NET Core 2.0
+- Mono 5.4
+- Xamarin.Android 8.0
+- Xamarin.Mac 3.8
+- Xamarin.iOS 10.14
+- Universal Windows Platform 10.0.16299
 
-The `ISentryClient` exposes the `CaptureEvent` method and its implementation `SentryClient` is responsible for queueing the event to be sent to Sentry. It also abstracts away the internal transport.
+## Unit Testing
```

Comment:
I wouldn't mix the information about where we run our unit tests with the documentation about how to do Unit Testing with our SDK.

This first part is just about giving confidence to users about how compatible our SDK is with different versions and runtimes. The second one is educating users on how they can use Sentry in their code, but still write proper unit tests.

Reply:
Got it. Thanks for clarifying.